.gitattributes

```diff
@@ -1,2 +1,3 @@
 
-common/test-fixtures/root/*	eol=lf
\ No newline at end of file
+common/test-fixtures/root/*	eol=lf
+*.go	eol=lf
```

Comment:
Is this relevant to the rest of the PR or left in from something else?

Reply:
I have a separate PR #4494 that deals with a lot more EOL/Windows issues. Technically no EOL is an error even if the *.go line is gratuitous. reverted.